lib/testLauncher/SuitestLauncher.js

```diff
@@ -128,14 +133,14 @@ class SuitestLauncher {
 					};
 
 					if (inspectBrk || inspect) {
-						if (this.restArgs[0] === process.execPath) {
-							this.restArgs.shift();
-						}
+						this.restArgs.shift();
 						cp.fork(this.restArgs[0], this.restArgs.slice(1), {
 							...options,
 							execArgv: [
 								`--inspect${inspectBrk ? '-brk' : ''}=${inspectBrk || inspect}`,
 							],
+							cwd: `${process.cwd()}`,
```

Comment:
@AntonSBilan why process.cwd() wrapped?

Reply:
Not sure how it works on windows environment, I thought about potential problem with white spaces in path

Reply:
I believe that will not converted